packages/terra-dropdown-button/docs/DropdownButton.md

```diff
@@ -0,0 +1,15 @@
+# Terra Dropdown Button
+
+The Terra Dropdown Button is a button that, when pressed, opens a dropdown with more buttons.
```

Comment:
Can we describe the use cases for this so we can differentiate from split button?

Reply:
@jenniferwurth has provided some wording for this on the GitHub issue: https://github.com/cerner/terra-core/issues/2216#issuecomment-501327457 along with wording for the split button.

Reply:
I'm not sure I like her wording for the DropdownButton, it seems to be in a more specific context than I'd say the documentation should be.